.github/CODEOWNERS

```diff
@@ -421,16 +502,108 @@
 /packages/grafana-o11y-ds-frontend/src/TraceToLogs/ @grafana/observability-traces-and-profiling
 /packages/grafana-o11y-ds-frontend/src/TraceToMetrics/ @grafana/observability-traces-and-profiling
 /packages/grafana-o11y-ds-frontend/src/TraceToProfiles/ @grafana/observability-traces-and-profiling
+
+# @grafana/plugin-configs
 /packages/grafana-plugin-configs/ @grafana/plugins-platform-frontend
+
+# @grafana/prometheus
 /packages/grafana-prometheus/ @grafana/oss-big-tent
-/packages/grafana-schema/src/**/*canvas* @grafana/dataviz-squad
-/packages/grafana-schema/src/**/*tempo* @grafana/observability-traces-and-profiling
+
+# @grafana/runtime
+/packages/grafana-runtime/CHANGELOG.md @grafana/grafana-frontend-platform
+/packages/grafana-runtime/LICENSE_APACHE2 @grafana/grafana-frontend-platform
+/packages/grafana-runtime/README.md @grafana/grafana-frontend-platform
+/packages/grafana-runtime/package.json @grafana/grafana-frontend-platform
+/packages/grafana-runtime/project.json @grafana/frontend-ops
+/packages/grafana-runtime/rollup.config.ts @grafana/frontend-ops
+/packages/grafana-runtime/src/index.ts @grafana/grafana-frontend-platform
```

Comment:
I'm wondering if it's worth having `@grafana/plugins-platform-frontend` here so we can at least keep tabs on what is being exposed to plugins? I'd probably add us to data and runtime too. 🤔